content/post/104-js-ipfs-0.48.md

```diff
@@ -0,0 +1,346 @@
+---
+date: 2020-07-20
+url: /2020-07-20-js-ipfs-0-48/
+title: js-IPFS 0.48.0 released with connectivity improvements and smaller blockstore
+header_image: js-ipfs-placeholder.png
+author: Alex Potsides
+---
+
+# 🔦 Highlights
+
+> delegates turned on by default, smaller, faster blockstore and a more intuitive API
+
+`js-IPFS@0.48.0` is hot off the press with much better default connectivity, a smaller blockstore and a more intuitive API
+
+## 🧭 Delegate nodes on by default
+
+JS IPFS has traditionally primarily targeted the browser, and the browser is a bad place to be if you want to be on the [DHT][]. You typically aren't on a page long enough to make or respond to [DHT][] queries and nor are you diallable so even if you were able to advertise yourself as a provider for a given block, the chances are no-one can connect to you to retrieve that block which results in a degraded service for everyone. Worse, the way you find more peers and content is via the [DHT][] so you're kind of stuck.
+
+There are several ways to give in-browser IPFS nodes a better experience on the network, one of those is [Delegate Nodes](https://blog.ipfs.io/2019-08-06-js-ipfs-0-37/#delegated-peer-and-content-routing). A Delegate Node is a network peer that performs certain actions on behalf of other nodes on the network. In this case it will make [DHT][] queries on our behalf so we can find more peers and more content than ever before.
+
+`js-IPFS@0.48.0` enables delegate nodes in the configuration by default. That means you should see far more peers that you have previously and be able to find content faster and more reliably than ever before.
+
```

Comment:
We should mention that the default config uses public delegate nodes, but in a production environment other delegate nodes should be setup to guarantee availability and avoid congestion. I have in my TODO list to create some documentation around libp2p infrastructure/deployment guidelines [libp2p/js-libp2p#696](https://github.com/libp2p/js-libp2p/issues/696) that will be useful for this. But meanwhile, I think we can just say that people should configure their own as we already have some information on how to do that, but it is spread across multiple channels

Reply:
Done!